src/model/user-settings/UserSettings.ts

```diff
@@ -446,9 +450,8 @@ export class UserSettings implements IUserSettings {
         "#container-view-settings"
       ) as HTMLDivElement;
     if ((await this.getProperty(ReadiumCSS.SCROLL_KEY)) != null) {
-      (
-        await this.getProperty(ReadiumCSS.SCROLL_KEY)
-      ).value = this.verticalScroll;
+      (await this.getProperty(ReadiumCSS.SCROLL_KEY)).value =
+        this.verticalScroll;
```

Comment:
I'm not totally sure what this is supposed to do. You await the value and then set it to `this.verticalScroll`, but I don't think that will change the value in the store will it?

Reply:
i'll double check that

Reply:
you are right, good catch, that doesn't do anything. i'll change it to this:

```  
    if ((await this.getProperty(ReadiumCSS.SCROLL_KEY)) == null) {
      await this.saveProperty(
        new Switchable(
          "readium-scroll-on",
          "readium-scroll-off",
          this.verticalScroll,
          ReadiumCSS.SCROLL_REF,
          ReadiumCSS.SCROLL_KEY
        )
      );
    }
```
that should work.

Reply:
removed this altogether